### PR TITLE
Create 2021-77.md

### PR DESCRIPTION
**Pull request by billfox1**
Release Notes for 8.11.21

### CODE REVIEW

**Review by janettelynch — APPROVED** [review]

File: release-notes/2021-77.md [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+ 
+- Resolved an issue where fields including the word `tag` showed the tag icon in the column header.
+
+- Resolved an issue where the recurring review on a compliance requirement did not send a reminder when the review was almost due.
+
+- Resolved an issue that occurred where the modal was not cleared between evidence uploads.
+
+## Coming Soon!
+
+- Update to Query Language: DeMorgan's Law:
```
Comment (erkangz) [inline release-notes/2021-77.md:117]: not a fan of the two colons back to back

```suggestion
- Update to Query Language following De Morgan's Law:
```

File: release-notes/2021-77.md [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+To maintain language correctness, J1QL will fulfill shorthand filters in accordance with DeMorgan's Law. This improvement only impacts queries that use the operators `!=`, `!~=`, `!^=`, `!$=` when operating on a group of values such as: `FIND jira_user WITH accountType != ('atlassian' OR 'app' OR 'customer')`. To put it simply, J1QL will interpret the above query to return all jira_user entities, excluding those that have an accountType value of 'atlassian' or 'app'. JupiterOne is taking precautions to ensure saved questions and queries are not inadvertantly effected, however if you have stored queries outside of JupiterOne, please be aware that this change may affect your query.
```
Comment (erkangz) [inline release-notes/2021-77.md:118]: empty line and two spaces needed before this. There's a typo. And the example can be a bit more clear.

```suggestion

  To maintain language correctness, J1QL will fulfill shorthand filters in accordance 
  with De Morgan's Law. This improvement only impacts queries that use the 
  operators `!=`, `!~=`, `!^=`, `!$=` when operating on a group of values.
  
  For example, 
   
  ```j1ql
  FIND jira_user WITH accountType != ('atlassian' OR 'app' OR 'customer')
  ```

  is the equivalent of

  ```j1ql
  FIND jira_user WITH 
    accountType != 'atlassian' AND 
    accountType !=  'app' AND 
    accountType !=  'customer'
  ```
   
  In other words, J1QL will interpret the above query to return all jira_user entities, 
  excluding those that have an accountType value of 'atlassian' or 'app' or 'customer'. 
   
  !!! warning
      **This is a breaking change!** We are taking precautions to ensure 
      saved questions and queries are not inadvertently effected. However, if you have stored 
      queries outside of your JupiterOne account (e.g. in a custom script), please update
      your query accordingly. 
   
  !!! note
      This change is planned to be rolled out on **Wednesday, Aug 18, 2021**.
```

File: release-notes/2021-77.md [hunk 3]
```diff
@@ -0,0 +1,122 @@
+# JupiterOne 2021.77 Release
+
+August 11, 2021
+
+## New Features and Improvements
+
+- Query Anywhere allows users to quickly access the query interface from any page in the product and get answers to their questions without switching contexts.
```
Comment (erkangz) [inline release-notes/2021-77.md:7]:
```suggestion
- New **Query Anywhere** feature that allows users to quickly access the query 
  interface from any page in the product and get answers to their questions without 
  switching contexts.
```

File: release-notes/2021-77.md [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+
+- Concatenation: The new J1QL return function `CONCAT()` allows users to combine two or more queried columns, strings, or numbers into one new string value.
+
+- Customers who configure an AWS integration for the first time are given the option to automatically import AWS CIS 1.4 (was previously 1.2).
```
Comment (erkangz) [inline release-notes/2021-77.md:11]: "AWS CIS 1.4" is not the correct reference. 

```suggestion
- Added **CIS AWS Benchmark v1.4** to the Compliance app. Customers who configure 
  an AWS integration for the first time will have v1.4 auto imported (was previously v1.2).
```

File: release-notes/2021-77.md [hunk 5]
```diff
@@ -0,0 +1,122 @@
...
+
+- There is a new **Request a feature** option in the JupiterOne help center dropdown menu, located above the **Report a bug option**.
```
Comment (erkangz) [inline release-notes/2021-77.md:13]:
```suggestion
- Added a new **Request a feature** option in the JupiterOne help center dropdown menu,
  located above the **Report a bug** option.
```

File: release-notes/2021-77.md [hunk 6]
```diff
@@ -0,0 +1,122 @@
...
+
+- There is improved styling to the Policies app.
```
Comment (erkangz) [inline release-notes/2021-77.md:15]:
```suggestion
- Improved styling to the Policies app.
```

File: release-notes/2021-77.md [hunk 7]
```diff
@@ -0,0 +1,122 @@
...
+- Fixed `undefined undefined` in user jobs to show proper `status` and
+  `statusText`, such as `401 Unauthorized`.
+
+- Fixed `UPLOAD_ERROR`, `NOT_ALLOWED` caused by `_icon` property on app and
+  personal app entities. Removed the `_icon` property as it is not allowed to be
+  an underscore property.
+
+- Migrated to the latest SDK, leading to normalized property names due to schema
+  validations:
+  - `login_id` -> `loginId`
```
Comment (erkangz) [inline release-notes/2021-77.md:99]: line 

```suggestion

  - `login_id` -> `loginId`
```

File: release-notes/2021-77.md [hunk 8]
```diff
@@ -0,0 +1,122 @@
+# JupiterOne 2021.77 Release
+
+August 11, 2021
+
+## New Features and Improvements
+
+- Query Anywhere allows users to quickly access the query interface from any page in the product and get answers to their questions without switching contexts.
+
+- Concatenation: The new J1QL return function `CONCAT()` allows users to combine two or more queried columns, strings, or numbers into one new string value.
```
Comment (erkangz) [inline release-notes/2021-77.md:9]:
```suggestion
- Added new **string concatenation** function to J1QL. The new J1QL 
  return function `CONCAT()` allows users to combine two or more queried 
  columns, strings, or numbers into one new string value. 
  
  For example: `Find Person Return CONCAT(Person.lastName, ', ', Person.firstName)`
```

**Review by erkangz — CHANGES_REQUESTED** [review]

File: release-notes/2021-77.md [hunk 9]
```diff
@@ -0,0 +1,157 @@
...
+  FIND jira_user WITH 
+    accountType != 'atlassian' AND 
+    accountType !=  'app' AND 
+    accountType !=  'customer'
+  ```
+   
+  In other words, J1QL will interpret the above query to return all jira_user entities, 
+  excluding those that have an accountType value of 'atlassian' or 'app' or 'customer'. 
+   
+  !!! warning
```
Comment (erkangz) [inline release-notes/2021-77.md:150]: fyi, this syntax will create nice styling like this below, when published: 
image

Comment (erkangz) [reply release-notes/2021-77.md:150]: supported admonitions are `warning`, `note`, or `reference`

Comment (billfox1) [reply release-notes/2021-77.md:150]: Not showing in the md.  Are you referring to Zendesk publishing?

File: release-notes/2021-77.md [hunk 10]
```diff
@@ -0,0 +1,157 @@
...
+      **This is a breaking change!** We are taking precautions to ensure 
+      saved questions and queries are not inadvertently effected. However, if you have stored 
+      queries outside of your JupiterOne account (e.g. in a custom script), please update
+      your query accordingly.
```
Comment (erkangz) [inline release-notes/2021-77.md:154]:
```suggestion
      **This is a breaking change!** We are taking precautions to ensure 
      saved questions and queries are not inadvertently effected. 
      
      We will run maintenance jobs to update all saved queries in questions, 
      alerts, and Insights dashboard widgets during the rollout of this change.
      You do _not_ have to make those changes manually.
      
      However, if you have stored queries outside of your JupiterOne account 
      (e.g. in a custom script), please update those queries accordingly. 
```

File: release-notes/2021-77.md (hunk 9 above)
Comment (billfox1) [reply release-notes/2021-77.md:150]: md Preview, rather.

File: release-notes/2021-77.md [hunk 11]
```diff
@@ -0,0 +1,157 @@
...
+
+- New **Query Anywhere** feature that allows users to quickly access the query 
+  interface from any page in the product and get answers to their questions without 
+  switching contexts.
+
+- Added new **string concatenation** function to J1QL. The new J1QL 
+  return function `CONCAT()` allows users to combine two or more queried 
+  columns, strings, or numbers into one new string value. 
+  
+  For example: `Find Person Return CONCAT(Person.lastName, ', ', Person.firstName)`
```
Comment (erkangz) [inline release-notes/2021-77.md:15]:
````suggestion
  For example: 
  
  ```j1ql
  Find Person 
  Return CONCAT(Person.lastName, ', ', Person.firstName)
  ```
````

File: release-notes/2021-77.md (hunk 9 above)
Comment (erkangz) [reply release-notes/2021-77.md:157]: correct, it'll show only after publishing, not in preview

**Review by erkangz — APPROVED** [review]

**Review by AnneNielsenJ1 — APPROVED** [review]

**Pull request merged**